### PR TITLE
Move cohort upload link + pilot cleanup

### PR DESCRIPTION
**Pull request by benilovj**
* Move the cohort upload link under Programmes > [specific programme] > Cohort. This will need more work as the cohort management designs develop but it's now possible to actually navigate to this link.
* Remove the "Manage pilot" area completely
* Remove mentions of the pilot from a couple of feature specs

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -97,6 +89,10 @@
           controller: "vaccination_records/edit",
           action: "update_date_and_time"
     end
+
+    resource :cohort_list, as: :cohort, only: %i[new create] do
```
Comment (thomasleese) [inline config/routes.rb:93]:
```suggestion
    resource :cohort_import, as: :cohort, only: %i[new create] do
```

Could we name the controller to `CohortImportController` at the same time to match the renamed model? It's the last step to moving away from the old "cohort lists" naming.

**Review by thomasleese — COMMENTED** [review]
Looks good, it's something I had on my to do list so nice to get it done! Just one comment about renaming the controller.

File: config/routes.rb (hunk 1 above)
Comment (benilovj) [reply config/routes.rb]: done!

**Review by tvararu — APPROVED** [review]
Nice one!

**Pull request merged**